.github/workflows/xrpl-go-ci.yml

```diff
@@ -3,7 +3,7 @@ on:
   push:
     branches: [ main ]
   pull_request:
-    branches: [ main, 'v*' ]
+    branches: [ '**' ]
```

Comment:
What does `**` mean?

Reply:
Its a pattern to reference all branch names